wagtail/admin/edit_handlers.py

```diff
@@ -876,6 +876,21 @@ def render(self):
         panel = render_to_string(self.template, self.get_context())
         return panel
 
+
+class PageSubscriptionPanel(EditHandler):
```

Comment:
Since this is so minimal, could we just combine it with `CommentPanel` for now and split it out later if we get more notification types?

Reply:
Thanks! Fixed + squashed